src/Facets/AcrossFacet.sol

```diff
@@ -38,9 +45,14 @@ contract AcrossFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
 
     /// @notice Initialize the contract.
     /// @param _spokePool The contract address of the spoke pool on the source chain.
-    constructor(IAcrossSpokePool _spokePool, address _weth) {
+    constructor(
+        IAcrossSpokePool _spokePool,
+        address _weth,
+        uint256 _maxQuoteTime
+    ) {
         spokePool = _spokePool;
         weth = _weth;
+        getStorage().maxQuoteTime = _maxQuoteTime;
```

Comment:
storage should not be set in the facet's constructor.
It only changes the storage in the implementation address.

Reply:
Yep, updated.